app/controllers/unsubscriptions_controller.rb

```diff
@@ -49,4 +80,17 @@ def check_authenticated(subscription)
   def api
     EmailAlertFrontend.services(:email_alert_api)
   end
+
+  def redirect_to_unsubscribe_latest
+    redirect_to "/email/unsubscribe/#{id(@latest_subscription)}"
+  end
+
+  def redirect_to_manage_subscriptions
+    # Whilst it would be nice to direct to 'https://www.gov.uk/email/manage' instead
+    # (and treat the user as authenticated), this would open a security hole whereby
+    # if user A forwards an email to user B, then subsequently changes their
+    # frequency, user B could click on the old unsubscribe link and view user A's
+    # subscriptions. So DON'T change this line:
+    redirect_to 'https://www.gov.uk/email/authenticate'
```

Comment:
In environments that aren't production, it won't always be `www.gov.uk` that we want to direct users to (e.g. in staging it would be `www.staging.publishing.service.gov.uk`.  Would it be possible to use Plek to determine the base for this redirect?  IIRC `Plek.new.website_uri` returns the correct value.